### ISSUE TITLE
Class __init__() shown for class with metaclass abc.ABCMeta and abstractmethod

### ISSUE THREAD

**Issue by corykinney** [mitmproxy/pdoc#273]
I'm not sure if there is a way to control this client side, telling pdoc to not show the initialization method, but I would think abstract base classes are widely used enough that it would be nice if pdoc could handle it. Perhaps by checking to see if instantiating the class works without error, or checking if it's specifically an instance of ABC/has the metaclass ABCMeta with an abstractmethod defined. Let me know if this is a valid issue or not!

#### Problem Description
pdoc includes an initialization function for a class with the abc.ABCMeta metaclass which cannot be called.

#### Steps to reproduce the behavior:
If you have a source file foo.py:
```
from abc import ABCMeta, abstractmethod


class Foo(metaclass=ABCMeta):
    @abstractmethod
    def bar(self):
        pass

```
and if you generate documentation with `pdoc foo` then the generated documentation includes an entry for `Foo()`. Of course if you try to instantiate an object of the class with `Foo()` you get the error:
```
TypeError: Can't instantiate abstract class Foo with abstract methods bar
```

#### System Information
pdoc: 7.0.1
Python: 3.9.4
Platform: Windows-10-10.0.19041-SP0

**Comment by mhils** [comment]
Good point!

I have changed pdoc to not show a constructor for abstract base classes without a custom docstring on their constructor. If your abstract base class does define `__init__` with a custom docstring and you do not want to display that, you can override pdoc's behavior with a custom module template (see https://pdoc.dev/docs/pdoc.html#control-what-is-documented). I'll keep it included by default as there might be sensible use cases where you want to document API that is intended to be subclassed by users.

**Comment by corykinney** [comment]
Wow, that was fast! Thanks for implementing that. So as I understand from the change log, it'll be implemented in whatever release is after pdoc 7.1.0, correct?

**Comment by mhils** [comment]
Yes, exactly! The release process is quite automated, if you have urgent needs I'm happy to ship a release soon. You can of course install from GitHub in the meantime :)
